src/ChuckDeviceController/Controllers/DashboardController.cs

```diff
@@ -477,7 +477,7 @@ public async Task<IActionResult> AddInstance()
 
                 // TODO: Valid geofence names
 
-                if (minLevel > maxLevel || minLevel < 0 || minLevel > 40 || maxLevel < 0 || maxLevel > 40)
+                if (minLevel > maxLevel || minLevel == 0 || minLevel > 40 || maxLevel == 0 || maxLevel > 40)
```

Comment:
Why? This isn't valid, min/max level 0 is perfectly fine. Example: Completing tutorials on accounts continuously.